libs/convert/job.go

```diff
@@ -0,0 +1,35 @@
+package convert
```

Comment:
This package is also called `convert` and deals with similar things as `config/convert`. Can we consolidate or use naming that disambiguates between the two?